rst_files/version_control.rst

```diff
@@ -384,7 +392,7 @@ You'll see something like this
 .. figure:: /_static/figures/git-create-pr.png
     :scale: 60%
 
-This gives us a quick overview of the commits we want to merge in, as well as the end-to-end differences
+This gives us a quick overview of the commits we want to merge in, as well as the file differences
```

Comment:
I wanted to stress that these differences were "end-to-end," i.e. that they are path-independent/independent of the commit history. Maybe something like "final differences"? Dunno.

Reply:
I see what you mean, but not sure if end-to-end really works - maybe overall?

Reply:
I like overall.